suite-native/module-settings/src/slice.ts

```diff
@@ -14,6 +15,7 @@ type SliceState = {
 
 const initialState: AppSettingsState = {
     colorScheme: 'system',
+    currency: fiatCurrenciesMap.usd,
```

Comment:
let's always prefix it with `fiat` so it should be `fiatCurrency`

Reply:
https://github.com/trezor/trezor-suite/pull/5895/commits/088ccfb38beef6c123b6dd7a50d6ae4c2f120a07